CMakeModules/FindCLANG_FORMAT.cmake

```diff
@@ -44,11 +45,11 @@ if(CLANG_FORMAT_EXECUTABLE)
                   OUTPUT_VARIABLE clang_format_version
                   ERROR_QUIET OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-  if(clang_format_version MATCHES "^clang-format version .*")
+  if(clang_format_version MATCHES "^[^ ]*[ ]*clang-format version .*")
     # clang_format_version sample: "clang-format version 3.9.1-4ubuntu3~16.04.1
     # (tags/RELEASE_391/rc2)"
     string(REGEX
-           REPLACE "clang-format version ([.0-9]+).*"
+           REPLACE "^[^ ]*[ ]*clang-format version ([.0-9]+).*"
```

Comment:
```suggestion
           REPLACE "^.*clang-format version ([.0-9]+).*"
```

Reply:
the whole regex thing is broken atm. The `.*` will match anything and thus the rest of the regex will fail

Reply:
That is not right, with a middle pattern .* will only match the shortest pattern.